install/bin/decj.bat

```diff
@@ -2,17 +2,22 @@
 
 setLocal EnableDelayedExpansion
 
+set "BIN_PATH=%~dp0"
+pushd "%BIN_PATH%.."
+set "APP_PATH=%CD%"
+popd
+
 if defined CP_SHARED_JAR_DIR (
- for /R %CP_SHARED_JAR_DIR% %%a in (*.jar) do (
-   set CLASSPATH=!CLASSPATH!;%%a
+ for /R "%CP_SHARED_JAR_DIR%" %%a in (*.jar) do (
+   set "CLASSPATH=!CLASSPATH!;%%a"
  )
 )
 
-set CLASSPATH="$INSTALL_PATH/bin/opendcs.jar;$INSTALL_PATH/bin/hibernate.cfg.xml;
-for /R $INSTALL_PATH/dep %%a in (*.jar) do (
-  set CLASSPATH=!CLASSPATH!;%%a
-)
+set "CLASSPATH=%BIN_PATH%opendcs.jar;%BIN_PATH%hibernate.cfg.xml;"
```

Comment:
It looks like we're already setting the classpath on lines 10-13 and then here we completely ignore that and start the classpath fresh.

Or at least we could if that CP_SHARED_JAR_DIR is set.

@agilmore2 is that variable (CP_SHARED_JAR_DIR) used by any HDB users?

Reply:
Yes, that is a issue.  I didn't test with CP_SHARED_JAR_DIR.